Doc: Enhance comments for `reset` option (#1342)
vuejs/vuefire: docs/guide/options-api-realtime-data.md

</FirebaseExample>

> By default, VueFire **will not reset** the property. You can customize this behavior by providing a second argument, or options, to the `firestoreUnbind`/`rtdbUnbind`

Reviewer: ```suggestion
By default, VueFire **does not reset** a bound property but you can customize this behavior by providing a second argument to the `firestoreUnbind`/`rtdbUnbind`
```

<FirebaseExample>

```js

> // default behavior is to leave the property unchanged after unbinding

Reviewer: ```suggestion
// default behavior: leave the property unchanged after unbinding
```

this.$databaseUnbind('user')
// same as
this.$databaseUnbind('user', false)
// Afterwards this.user will retain its value, e.g. { name: 'Eduardo' }

// if you set the second parameter to `true`, the property will be reset to a standard value
// for references bound as primitives or objects, this standard value is `null`
this.$databaseUnbind('user', true)
// Afterwards, this.user === null.

// for references bound as arrays, this standard value is an empty array
this.$databaseUnbind('documents', true)
// this.documents === []

> // you can specify what value to reset the property to, using the function syntax

Reviewer: ```suggestion
// you can specify what value to reset the property to, using a function instead of `true`
```
